tests/publiccloud/run_ltp.pm

```diff
@@ -17,14 +17,15 @@ use warnings;
 use testapi;
 use utils;
 use repo_tools 'generate_version';
+use Mojo::UserAgent;
 
 sub wait_for_guestregister
 {
     my ($instance) = @_;
     my $retries = 20;
 
     for (my $loop = 0; $loop < $retries; $loop++) {
-        my $out = $instance->run_ssh_command(cmd => 'sudo systemctl is-active guestregister', proceed_on_failure => 1);
+        my $out = $instance->run_ssh_command(cmd => 'sudo systemctl is-active guestregister', timeout => 300, type_command => 1, proceed_on_failure => 1);
```

Comment:
I think it is a good idea to move https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7813/files#diff-8eff9f64c4a8410248c1ee15006eb96cR22 this function to `lib/publiccloud/instance` and use it here as well.
Lets do it in a other PR.